plugins/database/quota/src/main/java/org/apache/cloudstack/api/command/QuotaTariffListCmd.java

```diff
@@ -43,22 +43,22 @@ public class QuotaTariffListCmd extends BaseListCmd {
     @Inject
     QuotaResponseBuilder _responseBuilder;
 
-    @Parameter(name = ApiConstants.USAGE_TYPE, type = CommandType.INTEGER, required = false, description = "Usage type of the resource")
+    @Parameter(name = ApiConstants.USAGE_TYPE, type = CommandType.INTEGER, description = "Usage type of the resource")
```

Comment:
these have note become required, but the default is just removed !!?

Reply:
The default value of `required` is `false`; therefore, there is no need to inform it as `false`:

https://github.com/apache/cloudstack/blob/d8c7e34b383d6e6ab4d00f3a07e3b2ab467eca36/api/src/main/java/org/apache/cloudstack/api/Parameter.java#L35